src/css/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  height: 36px;
+  margin: 0 16px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--dark-text);
+}
+
+.card__code {
+  margin: 4px 16px 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--light-text);
+}
+
+.card__price {
+  padding: 0;
+  margin: 25px 16px 0;
+  display: flex;
+  justify-content: space-between;
+  width: calc(200px - 34px);
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 16px;
+  color: var(--light-text);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 16px;
+  color: var(--dark-text);
+}
+
+.card__buy-button {
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  cursor: pointer;
+
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  text-decoration: none;
+  color: #fff;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  border: 1px solid transparent;
+  border-radius: 5px;
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  -ms-border-radius: 5px;
+  -o-border-radius: 5px;
```

Comment:
underhood webpack can do it automatically

Reply:
removed